### ISSUE TITLE
Support decimals in temperatures

### ISSUE THREAD

**Issue by PerfectlyNormal** [joggs/home_assistant_ebeco#34]
Haven't seen a changelog for the API, but I'm quite sure these fields are new (to me at least).
`temperatureRoomDecimals` and `temperatureFloorDecimals`, which is a double instead of a integer-masking-as-a-double that we get in `temperatureRoom` and `temperatureFloor`.

Example from my thermostat:

```json
{
    "temperatureSet": 21.0,
    "temperatureFloor": 21.0,
    "temperatureRoom": 18.0,
    "temperatureFloorDecimals": 20.9,
    "temperatureRoomDecimals": 18.1
}
```

additionally, the OpenAPI documentation for setting the temperature says it supports doubles as well. Might not be as interesting, even if it works.

But reporting back more accurate temperatures might be nice to have

**Comment by danilsson** [comment]
Hi!

I'm running this on my HA and it works great (I only use floor readings though, not room). 

https://github.com/joggs/home_assistant_ebeco/pull/36

**Comment by PerfectlyNormal** [comment]
Fixed in 7214866
